app/vmselect/main.go

```diff
@@ -412,6 +412,16 @@ func selectHandler(qt *querytracer.Tracer, startTime time.Time, w http.ResponseW
 			return true
 		}
 		return true
+	case "prometheus/-/healthy":
```

Comment:
```suggestion
	case "prometheus/-/healthy", "prometheus/-/ready":
```
would it simplify the code?

Reply:
Yes, but it would be inconsistent with previous implementation: https://github.com/VictoriaMetrics/VictoriaMetrics/blob/21f6d84b4b2054981bd0b92d0130a032b4c92a46/lib/httpserver/httpserver.go#L383-L392